docs/adding-zones.md

```diff
@@ -1,15 +1,12 @@
-# Adding Zones to a MinIO Cluster
+# Adding Zones to a MinIO Cluster [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
 
-[![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
-[![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
-
-This document explains how to add zones to an existing MinIO Cluster with Operator. This document is only applicable to a MinIO Cluster created by MinIO Operator.
+This document explains how to add zones to an existing MinIO Cluster with Operator. This is only applicable to a MinIO Cluster created by MinIO Operator.
```

Comment:
There was a brief discussion in slack #FAQ about Cluster Vs Deployment.

Since Kubernetes Clusters are a thing, if we settle on MinIO Cluster we end up with clusters in clusters.

I'd prefer we go with "Deployment". So MinIO Deployment | Distributed MinIO Deployment | Standalone MinIO Deployment.

If there are no objections.

Reply:
makes sense